install-latest.sh

```diff
@@ -389,7 +397,37 @@ get_binary_name() {
   esac
   echo "$name"
 }
+# Check if one version is less than another.
+# Versions are expected to be of the for X.X.X where X is some integer.
+# 
+# Given version strings X1.X2.X3 and Y1.Y2.Y3:
+# A version is less than another if there exists an Xn/Yn pair where Xn < Yn where for every preceding Xn/Yn pair in the sequence Xn == Yn.
+# Example:
+# 2.1.4 < 3.1.4
+# 3.5.0 < 3.6.5
+# 3.5.5 < 3.5.15
+#
+# Returns 0 when the first version argument < the second.
+# Returns 1 otherwise.
+version_less_than() {
+    for i in 1 2 3
+    do
+        VERSION_PIECE1=$(echo "$1" | cut -d '.' -f $i)
+        VERSION_PIECE2=$(echo "$2" | cut -d '.' -f $i)    
+
+        if [ "$VERSION_PIECE1" -lt "$VERSION_PIECE2" ]
+        then
+            return 0
+        fi
+
+        if [ "$VERSION_PIECE1" -gt "$VERSION_PIECE2" ]
+        then
+            return 1
+        fi
+    done
 
+    return 1
```

Comment:
Will this return true if you compare `3.4.7` to `3.4.7`? I.e. is this `<=`?

Reply:
You're right, let me change that to be properly `<`.